community_images/consul/ironbank/image.yml

```diff
@@ -24,13 +24,7 @@ input_registry:
   account: ironbank
 repo_sets:
   - hashicorp/consul:
-      input_base_tag: "1.14."
-      output_repo: consul-ib
-  - hashicorp/consul:
-      input_base_tag: "1.13."
-      output_repo: consul-ib
-  - hashicorp/consul:
-      input_base_tag: "1.12."
+      input_base_tag: "1.17."
```

Comment:
For consul-ib, we need to make an exception, we have 3 tags and 3 different repo names.